src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23399.cs

```diff
@@ -0,0 +1,38 @@
+#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23399(TestDevice device) : _IssuesUITest(device)
+	{
+		public override string Issue => "Closing Modal While App is Backgrounded Fails"; 
+
+		[Test]
```

Comment:
```suggestion
		[Test]
		[Category(UITestCategories.Navigation)]
```

Reply:
added the category in a964d40c15626a76cfbc7738b18122de818b33a9